docs/sources/dashboards/variables/_index.md

```diff
@@ -63,3 +63,7 @@ The following dashboards in Grafana Play provide examples of template variables:
 
 - Variable drop-down lists are displayed in the order they are listed in the variable list in Dashboard settings.
 - Put the variables that you will change often at the top, so they will be shown first (far left on the dashboard).
+- By default, variables will not have a default value. This means that the topmost value in the dropdown will always be selected.
+  - If you want to pre-populate a variable with an empty value, you can use the following workaround:
+    1. Add the `Include All Option` to the variable.
+    2. Set the `Custom all value` to a value like `.+`.  
```

Comment:
I wasn't entirely sure if you meant preselected or something else. If that word doesn't convey what you meant, please feel free to change it. 

```suggestion
- By default, variables don't have a default value. This means that the topmost value in the drop-down is always preselected. If you want to pre-populate a variable with an empty value, you can use the following workaround in the variable settings:
    1. Select the **Include All Option** checkbox.
    2. In the **Custom all value** field, enter a value like `+`.  
```